formatTest/typeCheckedTests/expected_output/reasonComments.re

```diff
@@ -1,29 +1,39 @@
 3; /* - */
 
-3;
+3; /*-*/
```

Comment:
I think the root cause of this is we don't have a location information for the atom "3". I will address this in another diff.